core/base/inc/ROOT/RConfig.h

```diff
@@ -468,8 +468,8 @@
 # else
 #   define _R__DEPRECATED_LATER(REASON) __attribute__((deprecated(REASON)))
 # endif
-// #elif defined(_MSC_VER)
-// #   define _R__DEPRECATED_LATER(REASON) __declspec(deprecated(REASON))
+#elif defined(_MSC_VER)
```

Comment:
Please make the enabling on Windows its own commit.

Reply:
Thanks! Done!